e2e-tests/bundle/bundle-size.test.js

```diff
@@ -1,12 +1,14 @@
 import filesize from 'filesize';
 import fs from 'fs';
 
+const maxBundleSizeInKiloBytes = 6;
+
 describe('bundle size', () => {
-    it('paypal.browser.min.js should be less than 5 KB', () => {
+    it(`paypal.browser.min.js should be less than ${maxBundleSizeInKiloBytes} KB`, () => {
```

Comment:
I noticed this e2e test was wrong. It said it was checking for a 5 KB threshold but actually checked for 18 KB 🤭.

The actual minified size is 5.12 KB so I set the threshold to 6 KB for some breathing room.